WORKSPACE

```diff
@@ -110,6 +110,16 @@ node_repositories(
 
 yarn_install(
     name = "npm",
+    manual_build_file_contents = """
+filegroup(
```

Comment:
this is really surprising; add a comment saying why this is needed?

Reply:
👍